pkg/pipelinerunmetrics/metrics_test.go

```diff
@@ -78,7 +78,7 @@ func TestMetricsOnStore(t *testing.T) {
 	}
 
 	// We check that there's no change when incorrect config is passed
-	MetricsOnStore(logger)(config.GetMetricsConfigName(), &config.ArtifactBucket{})
+	MetricsOnStore(logger)(config.GetMetricsConfigName(), &config.Store{})
```

Comment:
just curious what's config.Store and how is it different from config.ArtifactBucket?